examples/widgets/card_collection.dart

```diff
@@ -47,6 +49,27 @@ class CardCollectionAppState extends State<CardCollectionApp> {
     });
   }
 
+  void _initFixedSizedCardModels() {
+    const int cardCount = 27;
```

Comment:
it would be neat to also demo the actual infinite scrolling feature where the cardCount is null.